### DISCUSSION TITLE
How to remove intro.md without error

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [facebook/docusaurus#8692] [Q&A]
I still want to use the autogenerated function of sidebars, but Intro.md must exist. I want to remove the intro.md file, but an error is reported that the intro id does not exist,  Where can I change the default configuration?

![image](https://user-images.githubusercontent.com/15354455/220511850-8b1a694a-92de-4199-8bb6-6b218a1ab039.png)

**Comment by slorber** [comment]
In Docusaurus config, there's a `themeConfig.navbar.items` element that references that intro doc. You should delete the navbar item or make it target another doc.

Created an issue so that newcomers do not encounter such error messages in the future: https://github.com/facebook/docusaurus/issues/8696

**Reply by Chasen-Zhang** [reply]
You are so amazing. @slorber i love you
